nebula/ui/components/VPNViewBase.qml

```diff
@@ -18,11 +18,10 @@ Item {
    property alias _viewContentData: viewContent.data
    property alias _interactive: vpnFlickable.interactive
 
-   height: parent.implicitHeight
```

Comment:
I don't think this patch caused it but since we're here - seeing a lot of these complaints: `Warning: qrc:/nebula/components/VPNViewBase.qml:21:4: TypeError: Cannot read property 'height' of null (VPNViewBase.qml:21)` so maybe we just add an `if (parent)` to cheer up the console.